core/src/main/java/de/danoeh/antennapod/core/receiver/MediaButtonReceiver.java

```diff
@@ -30,6 +31,12 @@ public void onReceive(Context context, Intent intent) {
 			Intent serviceIntent = new Intent(context, PlaybackService.class);
 			serviceIntent.putExtra(EXTRA_KEYCODE, event.getKeyCode());
 			serviceIntent.putExtra(EXTRA_SOURCE, event.getSource());
+			//detect if this is a hardware button press
+			if (event.getEventTime() > 0 || event.getDownTime() > 0) {
```

Comment:
Is this documented somewhere? Looks rather hacky to me. Who knows what different vendors do differently.